dojo/home/views.py

```diff
@@ -23,28 +23,26 @@ def home(request: HttpRequest) -> HttpResponse:
 
 
 def dashboard(request: HttpRequest) -> HttpResponse:
+    # No one wants a slow dashboard
     engagements = get_authorized_engagements(Permissions.Engagement_View).distinct()
-    findings = get_authorized_findings(Permissions.Finding_View).distinct()
-
-    findings = findings.filter(duplicate=False)
+    findings = get_authorized_findings(Permissions.Finding_View).filter(duplicate=False).distinct()\
+        .only('severity', 'id', 'created', 'mitigated').prefetch_related('risk_acceptance_set')
 
     engagement_count = engagements.filter(active=True).count()
 
     today = timezone.now().date()
+    beginning_date = today - timedelta(days=6)
+
+    finding_count = findings.filter(created__gte=beginning_date, created__lte=today).order_by().count()
+
+    mitigated_count = findings.filter(mitigated__gte=beginning_date, mitigated__lte=today).only('id').order_by().count()
```

Comment:
Does `.only('id').order_by()` make sense when we only count the mitigated findings?